### PR TITLE
Ndm fixes

### PR DESCRIPTION
**Pull request by kasiaturbiarz**

### CODE REVIEW

File: src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs [hunk 1]
```diff
@@ -110,6 +102,19 @@ public async Task<RefundClaimStatus> TryClaimRefundAsync(DepositDetails deposit,
                 : RefundClaimStatus.Unconfirmed(transactionHash);
         }
 
+        private void DisplayRefundInfo(DepositDetails deposit, ulong now)
+        {
+            var timeLeftToClaimRefund = deposit.GetTimeLeftToClaimRefund(now);
+            if (timeLeftToClaimRefund > 0 && _logger.IsInfo)
```
Comment (SebastianDremo) [inline src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs:108]: we don't mix logger logic with program logic -> make it into two ifs

File: src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs [hunk 2]
```diff
@@ -110,6 +102,19 @@ public async Task<RefundClaimStatus> TryClaimRefundAsync(DepositDetails deposit,
...
+            {
+                _logger.Info($"Time left to claim a refund: {timeLeftToClaimRefund} seconds.");
+            }
+            else if (timeLeftToClaimRefund == 0 && _logger.IsInfo)
```
Comment (SebastianDremo) [inline src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs:112]: same

**Review by SebastianDremo — APPROVED** [review]

File: src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs (hunk 1 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs]: true, true

File: src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs (hunk 2 above)
Comment (tkstanczak) [reply src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs]: yes please, may lead to serious errors

**Comment by kasiaturbiarz** [comment]
https://github.com/DemerzelSolutions/ndm/issues/110

**Pull request merged**